alacritty_terminal/src/term/mod.rs

```diff
@@ -2202,6 +2235,11 @@ mod tests {
         fn send_event(&self, _event: Event) {}
     }
 
+    struct Mock;
+    impl EventListener for Mock {
+        fn send_event(&self, _event: Event) {}
+    }
+
```

Comment:
This results in duplicate definitions and the code doesn't compile with this. The previous (same) definition is just above.
So this can be safely removed (and it compiles again).